src/App.tsx

```diff
@@ -26,70 +27,135 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
```

Comment:
The commented out function description should be removed as it is not relevant anymore. 🧹